src/main/java/core/basesyntax/service/CreatorOperations.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class CreatorOperations {
```

Comment:
currently you don't use this class anywhere in your project

let's here implement the logic of receiving the necessary FruitOperation implementation instance for each FruitDto